### PR TITLE
Fix kubernetes helm upgrade

### PR DESCRIPTION
**Pull request by ptrovatelli**
- helm upgrade was re-initializing the secrets that are stored inside of the defectdojo chart
- do not create those specific secrets by default

see comments at the end of https://github.com/DefectDojo/django-DefectDojo/pull/1752

NB: this also has some valuable corrections: https://github.com/DefectDojo/django-DefectDojo/pull/1881

### CODE REVIEW

File: KUBERNETES.md [hunk 1]
```diff
@@ -236,7 +250,12 @@ helm install \
   --set database=postgresql \
   --set postgresql.enabled=true \
   --set postgresql.replication.enabled=true \
-  --set postgresql.replication.slaveReplicas=3
+  --set postgresql.replication.slaveReplicas=3 \
+  --set createSecret=true \
+  --set createRabbitMqSecret=true \
+  --set createRedisSecret=true \
+  --set createMysqlSecret=true \
```
Comment (alles-klar) [inline KUBERNETES.md]: We don't need the mysql secret of we use postgresql. Same for Redis.

Comment (ptrovatelli) [reply KUBERNETES.md]: yes I know but it seemed more simple to just give the command that always works.

Comment (ptrovatelli) [reply KUBERNETES.md]: I have added the info in the doc. hopefully that will make travis happy too, not sure what happened with travis

**Comment by ptrovatelli** [comment]
force-pushed travis fix (needs the createsecret* flags)

**Comment by ptrovatelli** [comment]
it was working fine when upgrading using the same image, but when changing image i've found an issue with the initializer job (that needs being removed) and now with the postgresql password for some containers (celery worker, initializer). 

working on it...

File: travis/script.sh [hunk 2]
```diff
@@ -72,20 +72,22 @@ if [ -z "${TEST}" ]; then
...
-	  "
-	  ;;
+    rabbitmq)
+      HELM_BROKER_SETTINGS=" \
+        --set redis.enabled=false \
+        --set rabbitmq.enabled=true \
+        --set celery.broker=rabbitmq \
+        --set createRabbitMqSecret=true \
+    "
+    ;;
```
Comment (alles-klar) [inline travis/script.sh]: Maybe missing an indent here? The travis jobs looks different than before.

Comment (ptrovatelli) [reply travis/script.sh]: I have tried homogenizing the indents: was a mix of tabs and blanks, I have used only 2 blanks for indentation. bad idea? what looks different exactly?

Comment (alles-klar) [reply travis/script.sh]: I provided a PR, but it seems it has nothing todo with travis...

**Comment by alles-klar** [comment]
flake8 fails, but this is already fixed here: https://github.com/DefectDojo/django-DefectDojo/pull/1992 and synk should be allowed to fail. So for me it is ok now. You may merge my PR https://github.com/ptrovatelli/django-DefectDojo/pull/3 before merging to dev.

**Comment by ptrovatelli** [comment]
@DefectDojo/defectdojo-moderators we're gonna need another reviewer for helm/kubernetes :)

**Pull request merged**